src/main/java/uk/ac/ebi/biostudies/controller/Mining.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.ebi.biostudies.controller;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ebi.biostudies.service.TextMiningLinkUpdater;
+
+import static uk.ac.ebi.biostudies.api.util.Constants.JSON_UNICODE_MEDIA_TYPE;
+
+/**
+ * Temporary url to test text mining msg rcv
+ */
+
+@RestController
+@RequestMapping(value="/api/v1/mining")
+public class Mining {
+    private static final Logger LOGGER = LogManager.getLogger(Mining.class.getName());
+
+    @Autowired
+    TextMiningLinkUpdater textMiningLinkUpdater;
+    ObjectMapper mapper = new ObjectMapper();
+
+
+    @RequestMapping(value = "/mine", produces = JSON_UNICODE_MEDIA_TYPE, consumes = JSON_UNICODE_MEDIA_TYPE, method = RequestMethod.POST)
```

Comment:
Shouldn't this be `update` instead of `mine`?

Reply:
I will change it